src/SFA.DAS.EmployerFinance/Data/Scripts/Script0008CreateAccountPayeSchemes.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE [dbo].[AccountPayeSchemes]
+(
+  [Id] BIGINT NOT NULL IDENTITY,
```

Comment:
Could we use the surrogate ID from employer accounts?

Reply:
See reply to previous comment.